politeiad/config.go

```diff
@@ -8,6 +8,7 @@ package main
 import (
 	"encoding/base64"
 	"fmt"
+	gomime "mime"
```

Comment:
Small style nit, but I would leave the stdlib names alone and rename the politeiad mime package instead.  Something short and concise like `pdmime`.